lib/scripts/k2s/image/Build-Image.ps1

```diff
@@ -451,9 +481,13 @@ if ($Push) {
 # Cleanup inside VM
 if (!$Keep -and !$Windows) {
     Write-Log 'Cleaning up temporary disk space in control plane VM' -Console
-    Invoke-CmdOnControlPlaneViaSSHKey 'find ~/tmp/docker-build -exec chmod a+w {} \; ; rm -rf ~/tmp/docker-build'
+    #Invoke-CmdOnControlPlaneViaSSHKey 'find ~/tmp/docker-build -exec chmod a+w {} \; ; rm -rf ~/tmp/docker-build'
```

Comment:
Why is this commented out?

Reply:
I had disabled the folder deletion for debugging one issue. This should be uncommented.